docs/tutorials/fundamentals/viewer.md

```diff
@@ -94,54 +94,54 @@ The viewer is organized into a few key areas which are explained in the next sec
 
-* **View** allows you to toggle full screen, the menu bar, play, display axes, the scale bar, tooltips, and the activity dock. 
+    **Note:** In macOS, **Preferences** is under the napari menu.
 
-* **Window** allows you to open the integrated console, display the layer controls and layer list.  
 
-* **Plugins** allows you to install and manage plugins and displays a list of plugins that are currently installed.  
+* **View** allows you to toggle full screen, the menu bar, play, display axes, the scale bar, tooltips, and the activity dock.
 
-* **Help** contains the citation and about information.  
+* **Window** allows you to open the integrated console, display the layer controls and layer list.
 
-+++  
+* **Plugins** allows you to install and manage plugins and displays a list of plugins that are currently installed.
+
+* **Help** contains the citation and about information.
+
++++
 
 ### Canvas
 
 The **canvas** is in the center of the viewer and contains the visual display of the data passed to **napari**, including `Images`, `Points`, `Shapes`, and other supported data types. Under the hood, the canvas is a `vispy.scene.SceneCanvas` object which has built-in support for features such as zooming and panning. As `vispy` uses `OpenGL` and your graphics card, panning and zooming are highly performant. You can return to the original zoom level by clicking the `home` button in the viewer buttons panel.
 
 +++
 
-![image: pan and zoom with napari](../assets/tutorials/viewer_pan_zoom.webm)  
+![image: pan and zoom with napari](../assets/tutorials/viewer_pan_zoom.webm)
 
 +++
 
 <!-- I don't know why "(layer_list)=" is here. -->
-(layer_list)=  
-### Layer list  
+(layer_list)=
+### Layer list
 
 Layers are one of the basic napari objects. There are different layer types for `Image`, `Points`, `Shapes`, and other data types. They can be added to the viewer either programmatically or through the GUI. Once added, they populate the layer list located on the bottom left side of the canvas.
 
-The layer list contains one widget for each of the layers that have been added to the viewer and includes a `thumbnail` that shows a miniaturized version of the currently viewed data, a `name` that is an editable text box, a `visibility` button that can be toggled on or off to show or hide the layer, and an `icon` for the layer type.  
+The layer list contains one widget for each of the layers that have been added to the viewer and includes a `thumbnail` that shows a miniaturized version of the currently viewed data, a `name` that is an editable text box, a `visibility` button (eye icon) that can be toggled on or off to show or hide the layer, and an `icon` for the layer type. Note that you can Option/Alt-click on the `visibility` button to show *just* that one layer, hiding all others. If you then Option/Alt-click on the `visibility` button of a layer a second time, the visibility state of all layers will be restored.
```

Comment:
Not sure how this got here, the PR is flagged 0.5.0 (that's when the feature is to be released)
https://github.com/napari/docs/pull/121/files

Reply:
ir comes from [#196](https://github.com/napari/docs/pull/196/files#diff-648f35eb5f85b8494d2fa4e6a0d29edaaa329a5e89d1788415e39eb9d74c2d39)

Reply:
Ah, so David's auto formatter—I assume—tagged that paragraph which pulled in the whole thing and I didn't notice/realize when approving.

Reply:
Should I revert the whole changes in #196 in this file?

Reply:
Ok. It needs to be reverted manually.

Reply:
The actual content (few headings changed) is so minor, so I'd be fine with it.
Documenting a feature that's not in the release seems bad.